src/main/resources/interface-specification.yml

```diff
@@ -8132,6 +8287,29 @@ components:
           type: boolean
         activeDescriptor:
           $ref: '#/components/schemas/CompactDescriptor'
+    Clients:
+      type: object
+      properties:
+        results:
+          type: array
+          items:
+            $ref: '#/components/schemas/CompactClient'
+        pagination:
+          $ref: '#/components/schemas/Pagination'
+      required:
+        - results
+        - pagination    
+    CompactClient:
```

Comment:
I just noticed that a model `Client` already exists (it just has 1 more field).
We could rename that one to `CompactClient` and use it here